### PR TITLE
pam_systemd fixes

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

**Comment by evverx** [comment]
This pull request **fixes 1 alert** when merging 59646affb70b25146467215d91864dbc933b349c into f606cd16d3a09425bc14c8272da58b1bbe625ce9 - [view on LGTM.com](https://lgtm.com/projects/g/systemd/systemd/rev/pr-d1d101e9408e8b0695b9f6da6e61bfa1ba8f8974)

**fixed alerts:**

* 1 for FIXME comment

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by poettering** [comment]
Fixed the typo @mbiebl found

File: man/pam_systemd.xml [hunk 1]
```diff
@@ -84,40 +84,42 @@
...
+
+        <listitem><para>Takes a single, short identifier string for the desktop environment. This may be used to
+        indicate the session desktop used, where this applies and if this information is available. For example:
+        <literal>GNOME</literal>, or <literal>KDE</literal>. It is recommended to use the same identifiers and
+        capitalization as for <varname>$XDG_CURRENT_DESKTOP</varname>, as defined by the <ulink
+        url="http://standards.freedesktop.org/desktop-entry-spec/latest/">Desktop Entry
+        Specification</ulink>. (However, note that <varname>$XDG_SESSION_DESKTOP</varname> only takes a single item,
+        and not a colon-separated list like <varname>$XDG_CURRENT_DESKTOP</varname>.) See
+        <citerefentry><refentrytitle>sd_session_get_desktop</refentrytitle><manvolnum>3</manvolnum></citerefentry> for
+        further details.</para></listitem>
```
Comment (sourcejedi) [inline man/pam_systemd.xml]: I would add `The <varname>XDG_SESSION_DESKTOP</varname> environment variable takes precedence.`

Also it's a bit confusing, this is now moved under the `desktop=` option, but then it says `note that <varname>$XDG_SESSION_DESKTOP</varname> only takes a single item, and not a colon-separated list like <varname>$XDG_CURRENT_DESKTOP</varname>`.  Perhaps just say "note that this option takes ..."

**Comment by poettering** [comment]
Force pushed a new version with @sourcejedi suggestions addressed. thanks! ptal!

**Comment by sourcejedi** [comment]
Agreed.  One less FIXME, as the bot keeps reminding us :).

(Sorry for fiddling with the tags. I second-guessed it as unnecessary, when the submitter has commit rights).

**Comment by poettering** [comment]
@sourcejedi by all means, if you like the PR as a while, and think it's good to be merged, set the green label (if CI hasn't passed yet) or merge it (if it has). It doesn#t matter who submitted the PR.

**Comment by evverx** [comment]
> as the bot keeps reminding us :)

@sourcejedi I try to add a personal touch whenever I can. Not that it's much different from the bot :-)

File: README [hunk 2]
```diff
@@ -169,9 +169,7 @@ REQUIREMENTS:
         dependencies:
 
         util-linux >= v2.27.1 required
-        dbus >= 1.4.0 (strictly speaking optional, but recommended)
-                NOTE: If using dbus < 1.9.18, you should override the default
-                policy directory (--with-dbuspolicydir=/etc/dbus-1/system.d).
```
Comment (keszybz) [inline README]: I think the NOTE should stay

**Comment by poettering** [comment]
Force pushed a new version now. Only change is the one requested by @keszybz. Somehow in my head 1.9.14 was > 1.9.18…

**Comment by evverx** [comment]
This pull request **fixes 1 alert** when merging b115f33cc62602080b65b0040081c516917e5c47 into d68c0833eac6d46a3ed83f0ba7a049fe24291226 - [view on LGTM.com](https://lgtm.com/projects/g/systemd/systemd/rev/pr-415bdd958544159f7f8b518eb8bb19a47559c693)

**fixed alerts:**

* 1 for FIXME comment

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**